### PR TITLE
DOCS-1696: Getting Started MVD

### PR DESCRIPTION
**Pull request by Kris-MultiSafepay**
This commit contains the MVD for getting started, including a menu icon
and shortcode for on-page bottons.

Signed-off-by: Kris-MultiSafepay <kris.stallenberg@multisafepay.com>

### Basic
- We only accept documentation that is proved to work on our LIVE API
- At least 1 approval is needed before pull requests can be merged
- The article must match our Synonym word list. For external commits, a MultiSafepay employee will check
- US English must be applied
- Use bullets for numerations where the sort order doesn't matter. For chronological order, use numbers. This does not apply to changelogs
- Set link on email address.

### Text style
- Paths are written like: _Configuration → System → Plugin → MultiSafepay_
- Use single quotation marks in your Markdown code (example title: 'Title of the article')
- Use Italics for buttons, clicks, paths, etc. Do mind to close the italics before the end of a sentence followed by a period
- Numerations / bullets are punctuated with a period at the end of the numerations / bullets.
- Periods should be excluded for sentences that finish in an e-mail address (e.g. "..contact us at <example@example.com> "  )

### Images
- Screenshot recomended sizes: Width: 820px - Height: auto

### CODE REVIEW

File: content/guides/getting-started/_index.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+layout: 'single'
+weight: 10
+short_description: "Follow our step-by-step instructions from creating a 
+test account to processing your first payment."
+
+---
+
+Welcome to MultiSafepay. We break down barriers and remove obstacles for commerce by providing smart payment solutions. Our core services allow you to accept payments, manage transactions and stimulate online sales. On top of that, we provide a suite of tools to boost conversion rates, create the perfect checkout experience and gain financial insights.
+
+With this guide, you will process your first payment in five easy steps.
```
Comment (Atiq-MultiSafepay) [inline content/guides/getting-started/_index.md:18]: By following this guide, you will be able to process your first payment in five easy steps.

File: content/guides/getting-started/_index.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+{{< br >}}
+
+## Step 1: Create a test account
+To create a test account, follow these steps:
+1. Fill out the signup form to [create a free test account](https://testmerchant.multisafepay.com/signup)
+2. Open the e-mail containing your **security code**
```
Comment (Atiq-MultiSafepay) [inline content/guides/getting-started/_index.md:25]: e-mail = email
As mentioned in the EN Woordenlijst on Confluence

File: content/guides/getting-started/_index.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+3. [Log in](https://testmerchant.multisafepay.com/) using the **security code** provided in the e-mail
```
Comment (Atiq-MultiSafepay) [inline content/guides/getting-started/_index.md:26]: e-mail = email

File: content/guides/getting-started/_index.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+*Are you not sure which integration method best suits your business?  
+Our [integration specialists](mailto:integration@multisafepay.com) are happy to help you.*
+
+
+{{< br >}}
+
+
+## Step 4: Go live!
+When you are ready, you [sign up for a live account](https://merchant.multisafepay.com/signup) to start processing real transactions.
```
Comment (Atiq-MultiSafepay) [inline content/guides/getting-started/_index.md:94]: When you are ready, you can [sign up for a live account](https://merchant.multisafepay.com/signup) to begin processing real transactions.

File: content/guides/getting-started/_index.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+Because we are a [payment service provider](/faq/getting-started/glossary/#payment-service-provider-psp) and [acquirer](/faq/getting-started/glossary/#acquirer), we are legally required to perform extra checks, which require you to submit multiple forms. You can prepare [these documents](/faq/getting-started/how-do-i-know-which-documents-i-need-to-submit-and-which-data-to-post-on-my-website) in advance if you wish to speed up the onboarding process.
```
Comment (Atiq-MultiSafepay) [inline content/guides/getting-started/_index.md:96]: MultiSafepay is a [payment service provider](/faq/getting-started/glossary/#payment-service-provider-psp) and [acquirer](/faq/getting-started/glossary/#acquirer), therefore, we are legally required to perform extra checks, which require you to submit multiple forms. You can prepare [these documents](/faq/getting-started/how-do-i-know-which-documents-i-need-to-submit-and-which-data-to-post-on-my-website) in advance if you wish to speed up the onboarding process.

File: content/guides/getting-started/_index.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+
+*If you need help creating a live account, please reach out to our [sales team](mailto:sales@multisafepay.com)*
```
Comment (Atiq-MultiSafepay) [inline content/guides/getting-started/_index.md:98]: *If you require assistance creating a live account, please reach out to our [sales team](mailto:sales@multisafepay.com)*

**Review by Atiq-MultiSafepay — CHANGES_REQUESTED** [review]

File: content/guides/getting-started/_index.md (hunk 1 above)
Comment (Kris-MultiSafepay) [inline content/guides/getting-started/_index.md:18]: You can start processing payments in five easy steps:

Comment (Kris-MultiSafepay) [inline content/guides/getting-started/_index.md:18]:
```suggestion
You can start processing payments in five easy steps:
```

File: content/guides/getting-started/_index.md (hunk 2 above)
Comment (Kris-MultiSafepay) [inline content/guides/getting-started/_index.md:25]:
```suggestion
2. Open the email containing your **security code**
```

File: content/guides/getting-started/_index.md (hunk 3 above)
Comment (Kris-MultiSafepay) [inline content/guides/getting-started/_index.md:26]:
```suggestion
3. [Log in](https://testmerchant.multisafepay.com/) using the **security code** provided in the email
```

File: content/guides/getting-started/_index.md (hunk 4 above)
Comment (Kris-MultiSafepay) [inline content/guides/getting-started/_index.md:94]:
```suggestion
When you are ready, you [sign up for a live account](https://merchant.multisafepay.com/signup) to begin processing real transactions.
```

File: content/guides/getting-started/_index.md (hunk 5 above)
Comment (Kris-MultiSafepay) [inline content/guides/getting-started/_index.md:96]:
```suggestion
MultiSafepay is a [payment service provider](/faq/getting-started/glossary/#payment-service-provider-psp) and [acquirer](/faq/getting-started/glossary/#acquirer). Therefore, we are legally required to perform extra checks, which require you to submit multiple forms. You can [prepare these documents](/faq/getting-started/how-do-i-know-which-documents-i-need-to-submit-and-which-data-to-post-on-my-website) in advance if you wish to speed up the onboarding process.
```

File: content/guides/getting-started/_index.md (hunk 6 above)
Comment (Kris-MultiSafepay) [inline content/guides/getting-started/_index.md:98]:
```suggestion
*If you require assistance creating a live account, please reach out to our [sales team](mailto:sales@multisafepay.com)*
```

**Pull request merged**